sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -69,15 +69,18 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
         if (this.model.get('saved')) {
             this.model.save({saved: false},
                 {success: this.savedToast});
-
-            if (this.model.changedAttributes()) {
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                        "Feed Title": this.model.feed.get('title') });
-            }
-
         } else {
             this.model.save({saved: true},
                 {success: this.savedToast});
+
+            if (this.model.changedAttributes()) {
+                mixpanel.track("Saved", { "Item Title": this.model.get('title'),
+                                        "Feed Title": this.model.feed.get('title') })
+
+                if (!this.model.feed.get('subscribed')) {
+                    this.showSubscribeModal();
```

Comment:
Let's add an option to showSubscribeModal (i.e. {'source': 'saved'}) that adjusts the copy of the subscribe-modal to say either saved or upvoted.